src/controllers/itemDetails/index.js

```diff
@@ -439,8 +439,20 @@ function renderName(item, container, context) {
     if (parentNameHtml.length) {
         if (parentNameLast) {
             // Music
+            // Determine if there are over 10 artists in the album. If so, concat with 'and X others.'
+            const artists = parentNameHtml[0].split(' / ');
+            const otherArtistCount = artists.length - 10;
+            let newParentNameHtml = artists.slice(0, 10).join(' / ');
+            newParentNameHtml = `${newParentNameHtml} ${globalize.translate('AndOtherArtists', otherArtistCount)}</br>`;
+
             if (layoutManager.mobile) {
-                html = '<h3 class="parentName musicParentName">' + parentNameHtml.join('</br>') + '</h3>';
+                if (artists.length > 10) {
+                    html = '<h3 class="parentName musicParentName ">' + newParentNameHtml + '</h3>';
+                } else {
+                    html = '<h3 class="parentName musicParentName">' + parentNameHtml.join('</br>') + '</h3>';
+                }
+            } else if (artists.length > 10) {
+                html = '<h3 class="parentName musicParentName">' + newParentNameHtml + '</h3>';
```

Comment:
I'm not familiar with the styles here so maybe someone else with some more familiarity with the project can answer: Is the `focuscontainer-x` classname needed here like it's applied to the `h3` in the `else` below?

Reply:
It should probably be included. The `focuscontainer` class is used to manage focus for keyboards/remotes. `focuscontainer-x` would "trap" the focus in this element when navigating on the x axis (until the last focusable element in the container is reached).

Reply:
Should I mark this conversation as resolved?